client/src/helpers/index.js

```diff
@@ -0,0 +1,105 @@
+import { isEmpty } from 'lodash';
+/**
+ *
+ * @description Abstracts validation functions
+ * @class Helper
+ */
+class Helper {
+  /**
+   *
+   * @description Ensures Admin priviledges to perform certain operations
+   * @static
+   * @param {any} req
+   * @returns true or false
+   *
+   * @memberOf Helper
+   */
+  static isAdmin(req) {
+    if (req.decoded.data.role !== 'admin') {
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   *
+   * @description Ensures all fields are present when adding or updating book
+   * @static
+   * @param {any} req
+   * @returns true or false
+   *
+   * @memberOf Helper
+   */
+  static isDefined(req) {
+    if (Object.keys(req.body).length < 5) return false;
+    return true;
+  }
+
+  /**
+   *
+   * @description Ensures all inputs are validated
+   * @static
+   * @param {any} req
+   * @returns isValid and errors
+   *
+   * @memberOf Helper
+   */
+  static inputValidation(req) {
+    const errors = {};
+    for (let i = 0; i < 5; i += 1) {
+      const field = Object.values(req.body)[i];
+      if (field === (undefined || null || '') || /^\s+$/.test(field)) {
```

Comment:
What are you checking for here?

Reply:
I'm validating the input values when adding a book or modifying a book with respect to the database.

Reply:
What are valid values for this field?